.github/workflows/build-test-deploy.yaml

```diff
@@ -27,25 +27,6 @@ jobs:
       - name: Fails in order to indicate that pull request needs to be marked as ready to review and unit tests workflow needs to pass.
         run: exit 1
 
-  test:
```

Comment:
I think we ought to leave at least `make vet`. we could add it to an existing action such as `test-integration`

Reply:
`go test` runs vet as step 1 regardless, iiuc.

Reply:
https://pkg.go.dev/cmd/go/internal/test "As part of building a test binary, go test runs go vet on the package
and its test source files" - though it does run a subset. 

The Makefile that is called with `make test` includes the `make vet` as a dependency, I'll add the same to `make test-integration`.